runner.py

```diff
@@ -60,6 +59,43 @@ def load_index():
     # init loading of query redactor
     config.query_redactor
 
+    if config.dev_config.pyroscope_url:
```

Comment:
could you please move it somewhere into `ols/src/utils/pyroscope` or so? TYVM!

Reply:
Moved it to `ols/utils/pyroscope` as suggested.